config/routes.rb

```diff
@@ -22,7 +22,7 @@
         get 'registracia-nastavenie', to: 'govbox#back_to_step1'
         get 'registracia-zabezpecenie', to: 'govbox#back_to_step1'
         get 'registracia-odoslanie', to: 'govbox#back_to_step1'
-        get 'ochrana-osobnych-udajov', to: 'static#privacy_policy'
+        get 'ochrana-osobnych-udajov', to: '/static#privacy_policy'
```

Comment:
preco?

Reply:
https://rollbar.com/SluzbySlovensko.Digital/www.ecosystem/items/17/

Reply:
cez `module: nil` alebo `module: ''`, pripadne iny option sa to neda spravit? to `/` mi pride ako typo na prvy pohlad

Reply:
Skusal som, nedalo sa ak to chceme mat na jednom mieste. Je to directory zapis, takze `/static` je ekvivalent `::StaticController`

Reply:
potom je to ok